backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -141,6 +162,12 @@ public String updatePatient(
 			residentCongregateSetting,
 			employedInHealthcare
 		);
+
+		if (patientToUpdate.getRole() != PersonRole.STAFF) {
```

Comment:
Same as above—I'd just check if facilityID is present. (Maybe pass it in as an `Optional`?)